Gemfile

```diff
@@ -44,6 +44,7 @@ gem 'bcrypt', '~> 3.1.7'
 # gem 'jbuilder', '~> 1.2'
 
 gem "bootstrap-sass", "3.3.3"
+gem "font-awesome-sass", "~> 4.4.0"
```

Comment:
My vote is to avoid adding Font Awesome for now. Let's limit the scope of this PR and replace icons in a later changeset.